client/src/components/v3/DASRecording.tsx

```diff
@@ -50,6 +53,7 @@ export default function DASRecording(): JSX.Element {
     setLoggingEnabled(false);
     stopLogging();
     stopBoost();
```

Comment:
Don't need these two because they call deprecated topics, same with `startBoost()` and `startLogging()`